### PR TITLE
Fixes in add-new-resource-short.md doc

### PR DESCRIPTION
**Pull request by turkenf**
### Description of your changes

This PR:

- Fixes broken links
- Update config method names

I have:

- [x] Read and followed Crossplane's [contribution process].
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [x] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### How has this code been tested

By following the doc

[contribution process]: https://git.io/fj2m9

### CODE REVIEW

File: docs/add-new-resource-short.md [hunk 1]
```diff
@@ -395,7 +395,7 @@ detailed guide could also help you.
 [provider-guide]:
     https://github.com/upbound/upjet/blob/main/docs/generating-a-provider.md
 [config-guide]:
-    https://github.com/upbound/upjet/blob/main/docs/configuring-a-resource.md
+    https://github.com/upbound/upjet/blob/main/docs/add-new-resource-long.md
 [issue-258]: https://github.com/upbound/official-providers/issues/258
```
Comment (turkenf) [inline docs/add-new-resource-short.md:399]: This link is also broken but I cannot solve it.

Comment (turkenf) [reply docs/add-new-resource-short.md:399]: Similarly there is another one pointing to https://github.com/upbound/official-providers/issues/new/choose

Comment (sergenyalcin) [reply docs/add-new-resource-short.md:399]: We may use the following link to solve this problem: `https://github.com/upbound/provider-aws/issues/90`

**Review by sergenyalcin — COMMENTED** [review]
@turkenf Thank you for this contribution. This is very much appreciated. I left a response to your comment that is related the unresolved broken link. We can change the non-working issue link with the new one that is in my comment.

**Comment by sergenyalcin** [comment]
> [CLA assistant check](https://cla.upbound.io/upbound/upjet?pullRequest=117) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.upbound.io/upbound/upjet?pullRequest=117) before we can accept your contribution.You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.upbound.io/check/upbound/upjet?pullRequest=117) it.

@turkenf Could you please sign the CLA?

**Review by sergenyalcin — APPROVED** [review]
Thanks @turkenf LGTM!

**Pull request merged**